01-Login/exec.ps1

```diff
@@ -1,2 +1,2 @@
 docker build -t auth0-servlet-webapp .
-docker run -p 8080:8080 -it -v ~/.gradle:/home/gradle/.gradle auth0-servlet-webapp
+docker run -p 3000:3000 -it -v ~/.gradle:/home/gradle/.gradle auth0-servlet-webapp
```

Comment:
this -v option is mounting the local directory `~./gradle` into the container folder `/home/gradle/.gradle`. Hosts should not need to install anything other than Docker.

Reply:
The gradle image is defining a volume in `/home/gradle/.gradle` directory, if we don't mount the volume dependencies installation won't be cached.